benchmarks/scala-stdlib/cafesat/build.sbt

```diff
@@ -0,0 +1,41 @@
+lazy val cafesat = taskKey[File]("Create the main run script")
+
+lazy val scalaSMTLib = RootProject(uri("../scala-smtlib"))
+
+lazy val runnerScriptTemplate =
+  """#!/bin/sh
+java -classpath "%s" %s "$@"
+"""
+
+exportJars := true
+
+cafesat := {
+  val cp = (fullClasspath in Runtime).value
+  val mainClass = "cafesat.Main"
+  val contents = runnerScriptTemplate.format(cp.files.absString, mainClass)
+  val out = target.value / "cafesat"
+  IO.write(out, contents)
+  out.setExecutable(true)
+  out
+}
+
+lazy val root = (project in file("."))
+  .settings(
+    name := "CafeSat",
+    version := "0.01",
+    scalaVersion := "2.12.8",
+    scalacOptions ++= Seq("-unchecked", "-deprecation", "-feature"),
+    javaOptions in IntegrationTest ++= Seq("-Xss10M"),
+    fork in IntegrationTest := true,
+    logBuffered in IntegrationTest := false,
+    parallelExecution in Test := true,
+    libraryDependencies += "org.scalatest" % "scalatest_2.11" % "2.2.1" % "test"
```

Comment:
Dependency on `scalatest` 2.11 is weird, but since we're only running the main artifact, it shouldn't be important.

Reply:
Might be a left over from some changes I did when pulling in the sources. I will check.

Reply:
Might be that CafeSAT had been using Scala 2.11.